bin/build_package.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * This file is used to compile the assets from an UX package.
+ */
+
+const { parseArgs } = require('node:util');
+const path = require('node:path');
+const fs = require('node:fs');
+const glob = require('glob');
+const rollup = require('rollup');
+const CleanCSS = require('clean-css');
+const { getRollupConfiguration } = require('./rollup');
+
+const args = parseArgs({
+    allowPositionals: true,
+    options: {
+        watch: {
+            type: 'boolean',
+            description: 'Watch the source files for changes and rebuild when necessary.',
+        },
+    },
+});
+
+async function main() {
+    const packageRoot = path.resolve(process.cwd(), args.positionals[0]);
+
+    if (!fs.existsSync(packageRoot)) {
+        console.error(`The package directory "${packageRoot}" does not exist.`);
+        process.exit(1);
+    }
+
+    if (!fs.existsSync(path.join(packageRoot, 'package.json'))) {
+        console.error(`The package directory "${packageRoot}" does not contain a package.json file.`);
+        process.exit(1);
+    }
+
+    const packageData = require(path.join(packageRoot, 'package.json'));
+    const packageName = packageData.name;
+    const srcDir = path.join(packageRoot, 'src');
+    const distDir = path.join(packageRoot, 'dist');
+
+    if (!fs.existsSync(srcDir)) {
+        console.error(`The package directory "${packageRoot}" does not contain a "src" directory.`);
+        process.exit(1);
+    }
+
+    if (fs.existsSync(distDir)) {
+        console.log(`Cleaning up the "${distDir}" directory...`);
+        await fs.promises.rm(distDir, { recursive: true });
+        await fs.promises.mkdir(distDir);
+    }
```

Comment:
This was not done in the previous script, but this is super useful to ensure we start with a clean `dist` directory.